### ISSUE TITLE
Claim list empty

### ISSUE THREAD

**Issue by niqofdez** [RoboMWM/ClaimlistClassifier#12]
Hey! I tried this plugin in MC 1.13.2 with GriefPrevention 16.11.6 but it doesn't seem to work. 
Do you require any specific version?
Image: https://i.imgur.com/QR69r1y.png

**Comment by RoboMWM** [comment]
No idea what version you're using or config or server version or anything.

**Comment by niqofdez** [comment]
Sorry! 
ClaimClassifier: The version published in bukkit (1.6 in plugin.yml)
Config: https://pastebin.com/c8f9P0rR
GriefPrevention 16.11.6 - Default config
Server version: Paper version git-Paper-641 (MC: 1.13.2) (Implementing API version 1.13.2-R0.1-SNAPSHOT)

**Comment by RoboMWM** [comment]
try disabling clickable claimslist?

**Comment by niqofdez** [comment]
Yes, with ClickableClaimslist false the claim list work, but this is the function that interests me.

**Comment by RoboMWM** [comment]
Ok thank you for testing. I'll put some debugging in.
